spec/unit/driverProviders/direct_test.js

```diff
@@ -57,9 +64,43 @@ describe('direct connect', function() {
       };
       var errorFound = false;
       try {
-        webdriver = require('../../../built/driverProviders/direct')(config);
+        webdriver = new Direct(config);
+        webdriver.getNewDriver();
+      } catch(e) {
+        errorFound = true;
+        expect(e.code).toBe(BrowserError.CODE);
+      }
+      expect(errorFound).toBe(true);
+    });
+  });
+
+  describe('binary does not exist', () => {
+    beforeEach((done) => {
+      // webdriver-manager clean
+      var argv = {
+        '_': ['update'],
+        'standalone': false,
+        'gecko': false
+      };
+      webdriverUpdate.run(JSON.parse(JSON.stringify(argv))).then(() => {
+        const SeleniumConfig = require('webdriver-manager/built/lib/config').Config;
+        rimraf.sync(path.resolve(SeleniumConfig.getSeleniumDir(), 'update-config.json'));
```

Comment:
Per @juliemr's recommendation, the best solution for now is probably to use a jasmine spy on `fs.readFileSync` and `fs.existsSync`